markdown/developers/pipeline_structure.md

```diff
@@ -0,0 +1,74 @@
+# nf-core contribution guidelines
+
+Nf-core pipelines adhere to common design principles that have been crafted through discussions and extensive real-world experiences with pipelines over time. We understand, however, that not all about nf-core pipelines is obvious and can be confusing to people new to nf-core. To help you along getting started with nf-core, writing your pipeline or contribute otherwise, we have created this documentation. Hopefully you will find most things that might confuse you described here, and if not, be sure to bug the nf-core Slack channel!
+
+## General
+
+The best way to write a new pipeline is to start from the template (see here). After doing that, you will have a number of files in your repository with different purposes. Here's what they are for:
```

Comment:
```suggestion
To start writing a new pipeline you should use the `nf-core create` command (see [docs](/developers/adding_pipelines)).
This will create a skeleton pipeline with a number of files, each with different purposes. Here's what they are for:
```